treelib/tree.py

```diff
@@ -296,6 +296,13 @@ def children(self, nid):
         """
         return [self[i] for i in self.is_branch(nid)]
```

Comment:
High, just a quick note - this method could now be implemented equivalently as `list(self.children(nid))`. That might address @caesar0301's comment re. redundancy.